src/main/java/pro/sky/telegrambot/controller/InfoController.java

```diff
@@ -24,8 +24,8 @@ public Info addInfo(@RequestBody Info newInfo) {
         return infoService.addInfo(newInfo);
     }
 
-    @PutMapping
-    public Info updateInfo(@PathVariable Long id,@RequestBody Info info) {
+    @PutMapping("/update/{id}")
```

Comment:
```suggestion
    @PutMapping("/{id}")
```
сам метод PUT уже говорит о том, что это update